web/src/components/newFeaturesSection.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import tw from 'twin.macro';
+
+import NewFeature from './newFeature';
+
+const StyledSection = styled.section`
+    ${tw`bg-gray-1 mx-auto pt-24 lg:pt-36 px-4 lg:px-2 2xl:px-0`}
+`;
+
+const StyledContainer = styled.div`
+    ${tw`container`}
+`;
+
+const LinkWrapper = styled.div`
+    ${tw`flex gap-10 justify-center mb-4`}
+`;
+
+const StyledButton = styled.button`
+    ${tw`text-gray-2 text-center font-semibold`}
+    color: ${({ isActive }) => (isActive ? '#00A09B' : 'bg-gray-2')}
```

Comment:
Styles don't match with figma. Check font-size and line-height. Also, active item should be underlined as well.